django/utils/functional.py

```diff
@@ -89,53 +89,33 @@ class __proxy__(Promise):
         until one of the methods on the result is called.
         """
 
-        __prepared = False
+        def __init__(self, args, kwargs):
+            self._args = args
+            self._kwargs = kwargs
```

Comment:
Renaming `__args` to `_args`, `__kw` to `_kwargs`, and the `kw` parameter, looks like unnecessary noise to me. Can we revert this?

Reply:
Yeah, will revert these.